### PR TITLE
clear-button

### PR DESCRIPTION
**Pull request by zeroadh**
clear button and _clearChit method on hole-page.js

### CODE REVIEW

File: src/components/hole-page.js [hunk 1]
```diff
@@ -83,6 +84,14 @@ class HolePage extends LitElement {
     // hole.chits = [];
     HoleService.updateHole(hole).then(savedHole=>this._getHole())
   }
+  
+  // Create _clearChit method
+  _clearChit(e, hole, chitId) {
+    hole.chits = hole.chits.filter(chit=>chit._id !=== chitID);
```
Comment (tylergraf) [inline src/components/hole-page.js]: Should be `chitId` here. (lowercase d)

Comment (tylergraf) [reply src/components/hole-page.js]: Also, should be `!==`.

**Review by tylergraf — CHANGES_REQUESTED** [review]

File: src/components/hole-page.js [hunk 2]
```diff
@@ -125,7 +134,9 @@ class HolePage extends LitElement {
             <input on-input="${e=>this._markChit(_hole, chit._chit._id, player._id)}" type="radio" name="${chit._chit._id}" checked="${chit._player && chit._player._id === player._id}" id="${chit._chit._id}_${player._id}">
             <label for$="${chit._chit._id}_${player._id}">${player.name}</label>
           </span>
+          
         `)}
+        <button on-click="${e=>this._clearChit(e, hole, chitId)}">Clear</button>
```
Comment (tylergraf) [inline src/components/hole-page.js]: Sorry, should be `_hole` and `chit._id` here.

**Review by tylergraf — CHANGES_REQUESTED** [review]

File: src/components/hole-page.js (hunk 2 above)
Comment (zeroadh) [reply src/components/hole-page.js]: Right...those should've been the actual variables passed as parameters in that method.  Sorry.  Fixed it.

**Comment by tylergraf** [comment]
Looks great!

**Pull request merged**